colossalai/cli/launcher/run.py

```diff
@@ -172,12 +173,14 @@ def _arg_dict_to_list(arg_dict):
                 value = extra_launch_args.pop(key)
                 default_torchrun_rdzv_args[key] = value
 
-        if torch_version.minor < 10:
+        if torch_version.major == 1 and torch_version.minor < 10:
+            # torch distributed launch cmd with torch == 1.9
```

Comment:
it would be better if you specify to '==' in this judgement

Reply:
Done